mumbleBot.py

```diff
@@ -71,7 +71,25 @@ def __init__(self, args):
             tt.daemon = True
             tt.start()
 
-        self.mumble = pymumble.Mumble(args.host, user=args.user, port=args.port, password=args.password,
+        if args.host:
+            host = args.host
+        else:
+            host = var.config.get("server", "host")
```

Comment:
Is the get raise an error if no value is present ?

Reply:
configuration.default.ini sets the host to 127.0.0.1, so that would raise a connection error at least if incorrect. I would blame the user if they remove the value without replacing it personally.

Reply:
Pymumble have weird reaction when the host isn't good.

Reply:
The situation currently doesn't seem to be any different. botamusique fails with
```
AttributeError: 'NoneType' object has no attribute 'comment'
```
on both connection refused and a totally incorrect IP (like "127.")